functions/success.js

```diff
@@ -0,0 +1,79 @@
+const environment = process.env.CONTEXT;
+
+const environmentKeys = {
+  production: {
+    STRIPE_KEY: process.env.STRIPE_SECRET_KEY,
+    WEBHOOK_KEY: process.env.STRIPE_WEBHOOK_SECRET,
+  },
+  other: {
+    STRIPE_KEY: process.env.STRIPE_TEST_KEY,
+    WEBHOOK_KEY: process.env.STRIPE_WEBHOOK_SECRET_TEST,
+  },
+};
+
+const apiKeys =
+  environment !== "production"
+    ? environmentKeys.other
+    : environmentKeys.production;
+
+const stripe = require("stripe")(apiKeys.STRIPE_KEY);
+const client = require('../utils/sanityClient');
+
+exports.handler = async function (event, context) {
```

Comment:
When is this `success` function called? I don't see a clear reference somewhere.

Reply:
There's a webhook in the Stripe setup that triggers the success function when a sale is complete. (there's also a CLI tool for triggering & listening to webhook events in development, to test that this works)